packages/structures/src/Mixin.ts

```diff
@@ -80,6 +83,10 @@ export function Mixin<DestinationClass extends typeof Structure<unknown>>(
 					if (typeof descriptor.value !== 'function') return;
```

Comment:
This should really skip not return, right?

Reply:
If that property exists while not being a function that’s an issue with the mixin itself. Should probably even throw here, not just return.